main/zoe/guide/README.md

```diff
@@ -60,9 +60,7 @@ that is escrowed with Zoe, Zoe guarantees that the user will either
 get back why they said they wanted, or the user will get back what they
 originally offered.
 
-When a user escrows with Zoe, they get two things back immediately: an escrow
-receipt, and a JavaScript promise for a future payout. This escrow
-receipt is what the user can send to smart contracts, as proof that they
+When a user escrows with Zoe, they get two things back immediately: a `seat`, and a JavaScript promise for a future payout. This `seat` is what the user can send to smart contracts, as proof that they
```

Comment:
Some of this was only true for the escrow receipt. Let's change this to something like: 
```suggestion
When a user escrows with Zoe, they get two things back immediately: a `seat`, and a JavaScript promise for a future payout. This `seat` has methods that the user can call to take action in the smart contract on Zoe...
```